nuopc_dart.F90

```diff
@@ -208,8 +208,8 @@ subroutine RealizeProvided(dgcomp, rc)
         file=__FILE__))&
         return ! bail out
   
-      call NUOPC_GetAttribute(field, "ConsumerTransferAction", &
-        transferAction, rc)  ! HK fudge, removed = on arguments to compile
+      call NUOPC_GetAttribute(field, name="ConsumerTransferAction", &
+        value=transferAction, rc=rc)  ! HK fudge, removed = on arguments to compile
```

Comment:
`=` assignment sign giving problem in compiling?

Reply:
This is the exact documentation which needs name and value to the function-
```
3.9.7 NUOPC_GetAttribute - Get the value of a NUOPC Field Attribute

INTERFACE:

   ! Private name; call using NUOPC_GetAttribute()
   subroutine NUOPC_GetAttributeFieldVal(field, name, value, isPresent, isSet, rc)
ARGUMENTS:
     type(ESMF_Field), intent(in)            :: field
     character(*),     intent(in)            :: name
     character(*),     intent(out)           :: value
     logical,          intent(out), optional :: isPresent
     logical,          intent(out), optional :: isSet
     integer,          intent(out), optional :: rc
DESCRIPTION:
Access the attribute name inside of field using the convention NUOPC and purpose Instance.

Unless isPresent and isSet are provided, return with error if the Attribute is not present or not set, respectively.

The arguments are:

field
The ESMF_Field object to be queried.
name
The name of the queried attribute.
value
The value of the queried attribute.
[isPresent]
Set to .true. if the queried attribute is present, .false. otherwise.
[isSet]
Set to .true. if the queried attribute is set, .false. otherwise.
[rc]
Return code; equals ESMF_SUCCESS if there are no errors.
```

Reply:
The problem was NUOPC_GetAttribute was missing from the use statement, (fixed in 52ee23cce54c848c3d4be38551b052a24153ceae)
the comment is outdated.